deployment/main.tf

```diff
@@ -135,3 +148,36 @@ output "acr_password" {
   sensitive = true
   value     = "${module.acr.password}"
 }
+
+output "client_cert" {
+  value     = "${module.tls.client_cert}"
+  sensitive = true
+}
+
+output "client_key" {
+  value     = "${module.tls.client_key}"
+  sensitive = true
+}
+
+output "cluster_ca" {
```

Comment:
Nit: cluster CA makes me think it's related to k8s cluster maybe should be 'ion_management_ca' or something similar?